guacamole/src/main/frontend/src/app/import/templates/connectionImportFileHelp.html

```diff
@@ -0,0 +1,93 @@
+<div class="import help">
+
+    <div class="header">
+        <h2>{{'IMPORT.SECTION_HEADER_HELP_CONNECTION_IMPORT_FILE' | translate}}</h2>
+        <guac-user-menu></guac-user-menu>
+    </div>
+
+    <h2>{{'IMPORT.HELP_FILE_TYPE_HEADER' | translate}}</h2>
+    <p>{{'IMPORT.HELP_FILE_TYPE_DESCRIPTION' | translate}}</p>
+
+    <h2>{{'IMPORT.HELP_CSV_HEADER' | translate}}</h2>
+    <p>{{'IMPORT.HELP_CSV_DESCRIPTION' | translate}}</p>
+    <p>{{'IMPORT.HELP_CSV_MORE_DETAILS' | translate}}</p>
+    <pre>name,protocol,hostname,group,users,groups,guacd-encryption (attribute)
+conn1,vnc,conn1.web.com,ROOT,guac user 1;guac user 2,Connection 1 Users,none
+conn2,rdp,conn2.web.com,ROOT/Parent Group,guac user 1,,ssl
+conn3,ssh,conn3.web.com,ROOT/Parent Group/Child Group,guac user 2;guac user 3,,
+conn4,kubernetes,,,,,</pre>
```

Comment:
Should it? I had assumed that this sort of example file content usually isn't. Plus doing all this formatting within the translation system seems like it'd be a pain - probably having to split this apart line by line.

Reply:
Since the example CSV itself contains English that's meant to be human-readable, I think it should be (if possible). Does angular-translate strip newlines from the translated text? If not, I'd think we could have something like:

    "IMPORT" : {
        "INFO_EXAMPLE_CSV" : "a,b,c\n1,2,3\n4,5,6"
    },

and:

    <pre>{{ 'IMPORT.INFO_EXAMPLE_CSV' | translate }}</pre>

Reply:
Adding `\n`s into the translation value in the JSON _does_ work! I'll switch all these over.